internal/support/auto-term.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# list taken from i3-sensible-terminal(1)
+terms="'$TERM' '$TERMINAL' terminal x-terminal-emulator xdg-terminal urxvt rxvt termit terminator Eterm aterm uxterm xterm gnome-terminal roxterm xfce4-terminal termite lxterminal mate-terminal terminology st qterminal lilyterm tilix terminix konsole kitty guake tilda alacritty hyper"
```

Comment:
The single quotes here are problematic because they're treated literally, meaning it tries to include them as part of the command.

I'd recommend removing $TERM and $TERMINAL from this list, then adding them directly to the for loop like so:
```
for term in "$TERM" "$TERMINAL" $terms; do
```
that'll provide the desired quoting behaviour while still handling any spaces in the value of those variables.